tests/Agent/IntegrationTests/ContainerApplications/SmokeTestApp/Dockerfile.amazon

```diff
@@ -0,0 +1,43 @@
+ARG TARGET_ARCH
+FROM --platform=${TARGET_ARCH} amazonlinux:latest AS base
+WORKDIR /app
+EXPOSE 80
+
+# build image is always amd64 (to match the runner architecture), even though the target architecture may be arm64
+FROM --platform=amd64 mcr.microsoft.com/dotnet/sdk AS build 
+ARG TARGET_ARCH
+WORKDIR /src
+COPY ["SmokeTestApp/SmokeTestApp.csproj", "SmokeTestApp/"]
+RUN dotnet restore "SmokeTestApp/SmokeTestApp.csproj"  -a ${TARGET_ARCH}
+
+COPY . .
+WORKDIR "/src/SmokeTestApp"
+RUN dotnet build "SmokeTestApp.csproj" -c Release -o /app/build --os linux -a ${TARGET_ARCH}
+
+FROM build AS publish
+RUN dotnet publish "SmokeTestApp.csproj" -c Release -o /app/publish /p:UseAppHost=false --os linux -a ${TARGET_ARCH}
+
+FROM base AS final
+
+# install asp.netcore - use the Centos 7 package feed because amazon is rhel based
+RUN rpm -Uvh https://packages.microsoft.com/config/centos/7/packages-microsoft-prod.rpm
+RUN yum install aspnetcore-runtime-7.0 -y
```

Comment:
Would it be worth making the runtime a parameter, in order to make transitioning to .NET 8 (etc) easier later?